src/more-infos/more-info-content.html

```diff
@@ -45,7 +45,7 @@
       }
     } else {
       window.hassUtil.dynamicContentUpdater(
-        this, 'MORE-INFO-' + window.hassUtil.stateMoreInfoType(stateObj).toUpperCase(),
+        this, 'more-info-' + window.hassUtil.stateMoreInfoType(stateObj).toLowerCase(),
```

Comment:
Can we make sure that `stateMoreInfoType` will always return lower case? Saves us a call.